spec/controllers/gift_requests_controller_spec.rb

```diff
@@ -89,5 +113,13 @@
       expect(@gift_requests.first.name).to eq(updated_gift_request[:name])
       expect(@gift_requests.first.description).to eq(updated_gift_request[:description])
     end
+
+    it "restricts non therapists from doing an update action" do
+      login_as_admin
+      put :update, id: @gift_requests.first.id, gift_request: {}
+      
+      expect(response).to redirect_to(root_path)
+      expect(assigns[:gift_request]).to be_nil
```

Comment:
You shouldn't be testing the internals of the controller method - make an update call that would update the gift request, then test that the gift request's attributes are unchanged.